lib/internal/Magento/Framework/Stdlib/DateTime/DateTime.php

```diff
@@ -141,6 +141,9 @@ public function timestamp($input = null)
                 $result = $input->getTimestamp();
                 break;
             default:
+				if (strpos($input, '/') !== false) {
+                    $input = str_replace('/', '-', $input);
```

Comment:
From https://stackoverflow.com/a/2891949 it looks like day and month will be mixed up by such change. Thoughts?

Reply:
Now, Day and Month will not be mixed with new changes. please review and test it again.

Reply:
Hello @orlangur 
i have added below code
if (strpos($input, '/') !== false && substr($input,0,2) > 12) {
$input = str_replace('/', '-', $input);
can you please retest it and give me your feedback.